mmv1/templates/terraform/examples/dns_managed_zone_private_multiproject.tf.erb

```diff
@@ -0,0 +1,106 @@
+resource "google_dns_managed_zone" "<%= ctx[:primary_resource_id] %>" {
+  name        = "<%= ctx[:vars]['zone_name'] %>"
+  dns_name    = "<%= ctx[:vars]['zone_dns_name'] %>"
+  description = "Example private DNS zone"
+  labels = {
+    foo = "bar"
+  }
+
+  visibility = "private"
+
+  private_visibility_config {
+    networks {
+      network_url = google_compute_network.network_1_project_1.id
+    }
+    networks {
+      network_url = google_compute_network.network_2_project_1.id
+    }
+    networks {
+      network_url = google_compute_network.network_1_project_2.id
+    }
+    networks {
+      network_url = google_compute_network.network_2_project_2.id
+    }
+  }
+
+  depends_on = [
+    google_project_service.compute_project_1,
+    google_project_service.dns_project_1,
+    google_project_service.compute_project_2,
+    google_project_service.dns_project_2,
+  ]
+}
+
+resource "google_project" "project_1" {
+  name            = "<%= ctx[:vars]['project_1_name'] %>"
+  project_id      = "<%= ctx[:vars]['project_1_name'] %>"
+  org_id          = "<%= ctx[:test_env_vars]['org_id'] %>"
+  billing_account = "<%= ctx[:test_env_vars]['billing_acct'] %>"
```

Comment:
Error is:

```
Final error: failed to enable services: failed on request preconditions: googleapi: Error 400: Billing account for project '151608109921' is not found. Billing must be enabled for activation of service(s) 'compute.googleapis.com,compute.googleapis.com,compute.googleapis.com' to proceed.
```

This is because the billing account is currently being set to an empty string: https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-8716-old..auto-pr-8716#diff-cb1c089ebc19fab4c219531c321c3fb56cf78a08e66785dacc89bb1490558deaR329

The key here (and in the other places the billing account is referenced) needs to match the key from the yaml file.

```suggestion
  billing_account = "<%= ctx[:test_env_vars]['billing_account'] %>"
```

Reply:
Thanks @melinath. Fixed. 👍